packages/manager/apps/container/src/container/nav-reshuffle/sidebar/index.tsx

```diff
@@ -402,159 +375,25 @@ const Sidebar = (): JSX.Element => {
-                  seeAllLabel={t('sidebar_pci_all')}
-                />
-                {pciError && (
-                  <button
-                    className={style.sidebar_pci_refresh}
-                    onClick={() => fetchPciProjects()}
-                  >
-                    <span>{t('sidebar_pci_load_error')}</span>
-                    <span className="oui-icon oui-icon-refresh"></span>
-                  </button>
-                )}
-                {selectedPciProject && (
-                  <span
-                    className={`flex px-1 ${style.sidebar_clipboard}`}
-                    title={t('sidebar_clipboard_copy')}
-                    onClick={() =>
-                      navigator.clipboard.writeText(
-                        selectedPciProject.project_id,
-                      )
-                    }
-                  >
-                    <span className={style.sidebar_clipboard_text}>
-                      {selectedPciProject.project_id}
-                    </span>
-
-                    <span
-                      className={`oui-icon oui-icon-copy px-1 mx-1  ml-auto ${style.sidebar_clipboard_copy}`}
-                    ></span>
-                  </span>
-                )}
-              </li>
-            )}
-
-            {menuItems?.map(({ node, count }) => (
-              <li
-                key={node.id}
-                id={node.id}
-                className={
-                  node === highlightedNode ? style.sidebar_selected : ''
-                }
-              >
-                {!shouldHideElement(node, count) && (
-                  <SidebarLink
-                    node={node}
-                    count={count}
-                    linkParams={{
-                      projectId: selectedPciProject?.project_id,
-                    }}
-                    onClick={() => menuClickHandler(node)}
-                    id={node.idAttr}
-                  />
-                )}
-                {node.separator && <hr />}
-              </li>
-            ))}
-          </ul>
-        )}
-      </div>
-
-      <Suspense fallback="">
-        <Assistance
-          containerURL={containerURL}
-          isOpen={isAssistanceOpen}
-          onToggle={(isOpen) => setIsAssistanceOpen(isOpen)}
-        />
-      </Suspense>
-    </div>
-  );
+  return <SidebarComponent 
+  logoLink={logoLink}
+  currentNavigationNode={currentNavigationNode}
+  trackingPlugin={trackingPlugin}
+  navigationPlugin={navigationPlugin}
+  servicesCount={servicesCount}
+  betaVersion={betaVersion}
+  pciError={pciError}
+  setIsAssistanceOpen={setIsAssistanceOpen}
+  fetchPciProjects={fetchPciProjects}
+  pciProjects={pciProjects}
+  selectedPciProject={selectedPciProject}
+  menuItems={menuItems}
+  menuClickHandler={menuClickHandler}
+  highlightedNode={highlightedNode}
+  containerURL={containerURL}
+  setCurrentNavigationNode={setCurrentNavigationNode}
+  isAssistanceOpen={isAssistanceOpen}
+  />
```

Comment:
that's a lots of props, navigation & tracking plugins could be retrieved using the shell context

some states could be retrieved using custom hooks & zustand for example 
```js
const { selectedPciProject, isAssistanceOpen, /* etc */ } = useSidebar();
```